lua/snacks/picker/config/sources.lua

```diff
@@ -352,6 +352,19 @@ M.grep = {
   show_empty = true,
   live = true, -- live grep by default
   supports_live = true,
```

Comment:
Hi @fang2hou, I believe you should additionally change the value of the toggle, otherwise the `R` flag is shown when regex is *dis*abled and it's not shown when regex is *en*abled.

```suggestion
  supports_live = true,
  toggles = {
    regex = { value = true },
  },
```

See the search result with your addition (notice no `R` is shown, but the dot `.`matches a parenthesis `(`:
<img width="727" height="201" alt="image" src="https://github.com/user-attachments/assets/c594c481-2d4e-41a0-8119-c0328bbc72d9" />

Now after toggling (disabling the default) regex the `R` is shown, but a literal `.` doesn't match the parenthesis:
<img width="708" height="115" alt="image" src="https://github.com/user-attachments/assets/2d6532d9-eda2-4761-b7aa-f3a65887c5e9" />